doc/man3/EVP_DigestInit.pod

```diff
@@ -387,7 +387,8 @@ B<EVP_MD>.
 =item EVP_MD_get_size()
 
 Return the size of the message digest when passed an B<EVP_MD> or an
-B<EVP_MD_CTX> structure, i.e. the size of the hash.
+B<EVP_MD_CTX> structure, i.e. the size of the hash. A negative value or 0 can
```

Comment:
Looking at the context - `or an B<EVP_MD_CTX>` should be removed as this is only about EVP_MD_get_size().